styles/pages/leaderboards/leaderboards.scss

```diff
@@ -157,3 +157,7 @@
 .leaderboard-dropdown {
 	vertical-align: center;
 }
+
+.leaderboard-endofrun-button {
+	margin-left: 8px;
+}
```

Comment:
Weird naming, this should be part of a BEM block (https://getbem.com/). I would probably put this in `leaderboards` so you have `leaderboards_button`.

I know I'm being pedantic about this but keeping consistent with CSS class naming is vital, stylesheets easily become a total mess if you don't follow a shared convention.